lib/bolognese/utils.rb

```diff
@@ -1133,6 +1133,100 @@ def hsh_to_spdx(hsh)
           "rightsIdentifierScheme" => hsh["rightsIdentifierScheme"],
           "schemeUri" => hsh["schemeUri"],
           "lang" => hsh["lang"] }.compact
+
+    def name_to_fos(name)
```

Comment:
I think this part is missing closing brackets. that's why the test fail

Reply:
I fixed this. Broke during merging the other pull request.